docs/docs/deploy/clever-cloud.md

```diff
@@ -0,0 +1,209 @@
+      "org_id": "<org>",
+      "deploy_url": "https://<some_src>.services.clever-cloud.com/app_<some_id>.git",
+      "name": "<app-name>",
+      "alias": "<app-name>"
+    }
+  ]
+}
+```
+
+:::note
+Note you **can** add this file to your git repository.
+:::
+
+:::tip
+By default, our runtime is scaled to **XS**, which is 1 CPU / 1Gio RAM, if you want less for the tests you can scale it down using.
+
+```bash
+clever scale --flavor nano
+```
+
+Giving the output `App rescaled successfully`
+:::
+
+### 2. Deploy your API to Clever Cloud
+
+As it works with `git` we do not send our binaries to them, the deployment on Clever Cloud is therefore in two steps
+
+    (0. send the sources)
+    1. build the app
+    2. run the app
+
+:::note
+_Clever Cloud_ listen for `8080` and this is great because dart_frog's is listening on 8080 to, so there is nothing to do with that, it will work out of the box.
+:::
+
+In order to tell _Clever Cloud_ how to build our project, we will need to create a `Dockerfile`, because the one created by `dart_frog build` is a built item and not sent to _Clever Cloud_ with _git_.
+
+Here is the `Dockerfile` you can add at the root of your project.
+
+```docker
+# This stage will compile sources to get the build folder
+FROM dart:stable AS build
+
+# Install the dart_frog cli from pub.dev
+RUN dart pub global activate dart_frog_cli
+
+# Set the working directory
+WORKDIR /app
+
+# Copy Dependencies in our working directory
+COPY pubspec.* /app/
+COPY routes /app/routes
+# Uncomment the following line if you are serving static files.
+# COPY --from=build public /app/public
+
+##### Add all of your custom directories here, for example if you have a "models" directory
+##### Add a line like this :
+# COPY models /app/models
+
+# Get dependencies
+RUN dart pub get
+
+# 📦 Create a production build
+RUN dart_frog build
+
+# Compile the server to get the executable
+RUN dart compile exe /app/build/bin/server.dart -o /app/build/bin/server
+
+
+# This stage build minimal serving image from AOT-compiled `/server` and required system
+# libraries and configuration files stored in `/runtime/` from the build stage.
+FROM scratch
+
+COPY --from=build /runtime/ /
+COPY --from=build /app/build/bin/server /app/bin/server
+# Uncomment the following line if you are serving static files.
+# COPY --from=build /app/build/public /public/
+
+# Expose the server port (useful for binding)
+EXPOSE 8080
+
+# Run the server
+CMD ["/app/bin/server"]
+```
+
+:::tip
+If you already have a `Dockerfile` you can rename it and specify to _Clever Cloud_ that the docker file to run is not `Dockerfile` using
+
+```bash
+clever env set CC_DOCKERFILE <the  name of your file>
+```
+
+:::
+
+Add the newly created `Dockerfile` to `git` via `git add Dockerfile`, commit and then run
+
+```bash
+clever deploy
+```
+
+Your **source code** will be sent to Clever Cloud.
+
+:::warning
+Consider Clever Cloud like a new origin and clever deploy a git push.
+:::
```

Comment:
Not sure what you mean with this?

Reply:
Its from a *git* point of view and I meant to say remote I'll edit this 
```suggestion
:::warning
Consider Clever Cloud like a new [remote (from a git point of view)](https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes) and clever deploy a git push.
:::
```

Reply:
Lgtm!